posthog/models/event/util.py

```diff
@@ -50,7 +92,19 @@ def create_event(
         "distinct_id": str(distinct_id),
         "elements_chain": elements_chain,
         "created_at": timestamp.strftime("%Y-%m-%d %H:%M:%S.%f"),
-        # TODO: Support persons on events
+        "person_id": str(person_id) if person_id else "00000000-0000-0000-0000-000000000000",
+        "person_properties": json.dumps(person_properties) if person_properties is not None else "",
+        "person_created_at": person_created_at.strftime("%Y-%m-%d %H:%M:%S.%f") if person_created_at is not None else 0,
+        "group0_properties": json.dumps(group0_properties) if group0_properties is not None else "",
+        "group1_properties": json.dumps(group1_properties) if group1_properties is not None else "",
+        "group2_properties": json.dumps(group2_properties) if group2_properties is not None else "",
+        "group3_properties": json.dumps(group3_properties) if group3_properties is not None else "",
+        "group4_properties": json.dumps(group4_properties) if group4_properties is not None else "",
+        "group0_created_at": group0_created_at.strftime("%Y-%m-%d %H:%M:%S.%f") if group0_created_at is not None else 0,
```

Comment:
this can't be None based on the code above, furthermore just use the function you created above here instead

Reply:
ah yup